packages/terra-functional-testing/src/commands/validates/index.js

```diff
@@ -0,0 +1,5 @@
+const accessibility = require('./accessibility');
```

Comment:
Just curious, why add the extra files to maintain instead of adding the src directly to index?

Reply:
Personal preference, in general I think it's bad practice to overextend a single file. The individual files make reading and testing easier/isolated/explicit and allows scaling as more commands are added